src/main/java/ProjectOcean/Controller/YearController.java

```diff
@@ -1,37 +1,49 @@
 package ProjectOcean.Controller;
 
+import ProjectOcean.Model.*;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
 import ProjectOcean.Model.CoursePlanningSystem;
 import ProjectOcean.Model.ICourse;
 import ProjectOcean.Model.IYear;
-import javafx.fxml.FXML;
-import javafx.fxml.FXMLLoader;
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
 import javafx.scene.input.DragEvent;
 import javafx.scene.input.TransferMode;
 import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Pane;
 import javafx.scene.layout.VBox;
 import java.io.IOException;
+import java.util.*;
```

Comment:
No need to import whole package. Could import HashMap, Map, Observable, List indiviadually.